packages/css/src/components/grid/grid.scss

```diff
@@ -3,11 +3,37 @@
  * Copyright (c) 2023 Gemeente Amsterdam
  */
 
+@use "sass:list";
+@use "sass:map";
```

Comment:
Using these functions creates kind of hard to follow loops, and it doesn't really reduce the amount of code. It's not really an improvement imo.